bin/setup

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env bash
+
+# ========== Install mysql ==========
+echo "Checking MySQL"
+
+# Check for existing mysql
+mysql_exists=`brew list -1 | grep mysql`
+if [ -z $mysql_exists ]; then
+    echo "  Installing MySQL"
+    brew install mysql
+fi
+# Check to see if mysql is running
+mysql_running=`ps ax | grep mysqld_safe | grep -v grep | awk '{print $1}'`
+if [ -z $mysql_running ]; then
+    echo "  Attempting to start MySQL"
+    brew services start mysql
+fi
+
+# Check to see if testing db is present
+mltshp_testing_db_exists=`mysql -u root -e "SELECT SCHEMA_NAME FROM INFORMATION_SCHEMA.SCHEMATA WHERE SCHEMA_NAME = 'mltshp_testing'" | tail -1`
+if [ -z $mltshp_testing_db_exists ]; then
+    echo "  Creating 'mltshp_testing' database"
+    mysql -u root -e "CREATE DATABASE mltshp_testing"
+fi
+
+# Check to see if development db is present
+mltshp_dev_db_exists=`mysql -u root -e "SELECT SCHEMA_NAME FROM INFORMATION_SCHEMA.SCHEMATA WHERE SCHEMA_NAME = 'mltshp'" | tail -1`
+if [ -z $mltshp_dev_db_exists ]; then
+    echo "  Creating and seeding 'mltshp' database"
+    mysql -u root -e "CREATE DATABASE mltshp"
```

Comment:
possibly catch errors and crash if these fail, by adding " || exit 1" after each one?